$NetBSD: patch-src_extensions_ocsipersist-sqlite_Makefile,v 1.2 2018/11/29 10:54:14 jaapb Exp $

The lwt_log package has been separated from lwt
Replace tyxml.parser with xml-light (patch from upstream)
--- src/extensions/ocsipersist-sqlite/Makefile.orig	2018-02-01 12:55:17.000000000 +0000
+++ src/extensions/ocsipersist-sqlite/Makefile
@@ -1,8 +1,9 @@
 include ../../../Makefile.config
 
-PACKAGE  := lwt.preemptive  \
-	    tyxml.parser \
+PACKAGE  := lwt.unix  \
+	    xml-light    \
 	    sqlite3      \
+			lwt_log
 
 LIBS     := -I ../../baselib -I ../../http -I ../../server \
 	    ${addprefix -package ,${PACKAGE}}
